routemaster/migrations/versions/e7d5ad06c0d1_add_updated_field_to_label.py

```diff
@@ -0,0 +1,31 @@
+"""
+add updated field to label
+
+Revision ID: e7d5ad06c0d1
+Revises: e1fec9622785
+"""
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = 'e7d5ad06c0d1'
+down_revision = 'e1fec9622785'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        'labels',
+        sa.Column(
+            'updated',
+            sa.DateTime(),
```

Comment:
Should this also be annotated that it's a timezone aware date-time?

Reply:
No, timezone awareness apparently does not apply to the table definition.